lib/src/screens/receive/receive_page.dart

```diff
@@ -53,7 +53,8 @@ class ReceivePage extends BasePage {
   final FocusNode _cryptoAmountFocus;
 
   @override
-  Color get titleColor => Colors.white;
+  Color? get titleColor =>
+      currentTheme.type == ThemeType.bright ? Colors.white : null;
```

Comment:
can be removed too, having it with null will use the correct color for each theme by default

Reply:
Removing the white color from `receive_page` makes it look like this on the `Bright` theme:
![image](https://user-images.githubusercontent.com/76502841/235918550-8f574be5-bd83-4054-a280-4efd5dac5d39.png)
Probably because the default color is meant for light background pages, like here:
![image](https://user-images.githubusercontent.com/76502841/235918686-2fc32374-4d94-4b28-8a8e-aaab7a1cddbe.png)
with only `Bright` having 2 different backgrounds depending on the page, white or gradient, so this theme in specific still needs a way to have 2 different colors